keyboards/minidox/keymaps/dustypomerleau/config.h

```diff
@@ -0,0 +1,30 @@
+// dustypomerleau, Minidox config
+
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
+
+#define EE_HANDS
+#define IGNORE_MOD_TAP_INTERRUPT
+#define ONESHOT_TIMEOUT 1000
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This is no longer needed, as it is now the default behavior.

Reply:
Thanks for the input! All changes accepted and committed.